build.gradle

```diff
@@ -1,12 +1,34 @@
 buildscript {
-  repositories { jcenter() }
+  repositories { 
+    jcenter()
+    mavenCentral()
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+  }
   dependencies {
-    classpath 'com.netflix.nebula:gradle-rxjava-project-plugin:4.0.0'
-    classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.1.0'
+    classpath 'ru.vyarus:gradle-animalsniffer-plugin:1.2.0'
+    classpath 'gradle.plugin.nl.javadude.gradle.plugins:license-gradle-plugin:0.13.1'
+    classpath "me.champeau.gradle:jmh-gradle-plugin:0.4.4"
+    classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
+    classpath "org.jfrog.buildinfo:build-info-extractor-gradle:latest.release"
```

Comment:
I'd strongly recommend to replace `latest.release` with particular version like `4.5.2`, otherwise build won't be reproducible

Reply:
Yeah, but these are snapshot/release builds which are by definition non-reproducible as there can be only one version.

Reply:
Well, non-reproducible also means that RxJava release might fail unpredictably because this plugin was updated even if no one changed anything Gradle-related in RxJava itself…

Reply:
Okay.